src/modules/map/components/LocationPopup.vue

```diff
@@ -1,17 +1,13 @@
 <template>
-    <div v-if="isRightClick" class="location-popup" data-cy="location-popup" @contextmenu.stop>
-        <div class="card">
-            <div class="card-header d-flex">
-                <span class="flex-grow-1 align-self-center">
-                    {{ $t('position') }}
-                </span>
-                <ButtonWithIcon
-                    data-cy="profile-popup-close-button"
-                    :button-font-awesome-icon="['fa', 'times']"
-                    @click="onClose"
-                />
-            </div>
-            <div class="card-body coordinates-list text-start">
+    <div v-if="isRightClick" class="location-popup" @contextmenu.stop>
```

Comment:
Side note. I tried to use a `ref` so I can add the HTML comment. But that didn't work. No idea what I did wrong :smile:

Reply:
that is perplexing...

Reply:
Indeed :smile: Try it when your working on the file anyway. It's very well possible that I just did some newbie mistake.